### ISSUE TITLE
When deleting an entry in the listview, KeepNote should automatically select the next note

### ISSUE THREAD

**Issue by GoogleCodeExporter** [fracte/keepnote#20]
```
If there are several entries in the listview, once an entry is
deleted by pressing the suppr key in the listview, the next entry of
the list should be automatically selected. At present time, i loose the
focus on the treeview if i delete an entry
```

Original issue reported on code.google.com by `thibaut....@gmail.com` on 4 Mar 2009 at 12:46

**Comment by GoogleCodeExporter** [comment]
```
The next release of KeepNote should have this feature.
```

Original comment by `matt.rasmus@gmail.com` on 10 Mar 2009 at 3:15
- Changed state: **Started**
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
This should now be available in 0.5.2.
```

Original comment by `matt.rasmus@gmail.com` on 14 Apr 2009 at 1:33
- Changed state: **Done**
